src/pages/components/Sidebar/SidebarContent.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Box, CloseButton, Flex, Text, useColorModeValue } from '@chakra-ui/react';
+import PropTypes from 'prop-types';
+import { AtSignIcon, ChatIcon, HamburgerIcon, StarIcon, WarningIcon } from '@chakra-ui/icons';
+import SidebarItem from './SidebarItem';
+
+const LinkItems = [
+  { name: 'Profile', icon: <AtSignIcon /> },
+  { name: 'Dashboard', icon: <ChatIcon /> },
+  { name: 'API Documentation', icon: <StarIcon /> },
+  { name: 'Contact Us', icon: <WarningIcon /> },
+];
+
+const SidebarContent = ({ onClose, ...rest }) => (
+  <Box
+    transition="3s ease"
+    bg={useColorModeValue('white', 'gray.900')}
+    borderRight="1px"
+    borderRightColor={useColorModeValue('gray.200', 'gray.700')}
+    w={{ base: 'full', md: 60 }}
+    pos="fixed"
+    h="full"
+    {...rest}
+  >
+    <Flex h="20" alignItems="center" mx="8" justifyContent="space-between">
+      <Text fontSize="2xl" fontFamily="monospace" fontWeight="bold">
+        IgboAPI
+      </Text>
+      <CloseButton display={{ base: 'flex', md: 'none' }} onClick={onClose} />
+    </Flex>
+
+    <SidebarItem
+      icon={<HamburgerIcon />}
+      _active={{ bg: 'gray.200' }}
+      _hover={{ bg: 'gray.200' }}
+      css={{
+        '&:hover': {
+          bg: 'inherit !important',
+        },
+      }}
```

Comment:
this styling should be baked into the SidebarItem component - this means that instead of this being a prop, it should just be inside the SidebarItem